.eslintrc.js

```diff
@@ -24,5 +24,6 @@ module.exports = {
       },
     ],
     'jsx-a11y/label-has-for': [2, { required: 'id' }],
+    'no-param-reassign': ["error", { "props": false }],
```

Comment:
What was the reason for this? In what case does it make sense to reassign a prop?

Reply:
Fixed in fe1c08403352a7e5c18fa8bb85b47b6e350bd476

Reply:
Ah great, the new method is way better.
Definitely good to avoid bypassing/mutating the component state.